beacon_node/src/config.rs

```diff
@@ -468,9 +468,19 @@ pub fn get_config<E: EthSpec>(
     client_config.chain.checkpoint_sync_url_timeout =
         clap_utils::parse_required::<u64>(cli_args, "checkpoint-sync-url-timeout")?;
 
-    client_config.genesis = if let Some(genesis_state_bytes) =
-        eth2_network_config.genesis_state_bytes.clone()
-    {
+    // If the `--genesis-state-url` is defined, use that to download the
+    // genesis state bytes. If it's not defined, try `--checkpoint-sync-url`.
+    let genesis_state_url_opt =
+        clap_utils::parse_optional::<String>(cli_args, "genesis-state-url")?;
+    let checkpoint_sync_url_opt =
+        clap_utils::parse_optional::<String>(cli_args, "checkpoint-sync-url")?;
+    client_config.genesis_state_url = genesis_state_url_opt.or(checkpoint_sync_url_opt);
+
+    client_config.genesis_state_url_timeout =
+        clap_utils::parse_required(cli_args, "genesis-state-url-timeout")
+            .map(Duration::from_secs)?;
```

Comment:
Just a note here - It would be possible for user to only specify "checkpoint-sync-url" and "checkpoint-sync-url-timeout" and get a timeout if checkpoint sync server doesn't respond with the genesis state within `genesis_state_url_timeout`. Not sure if we should change anything though.

Reply:
Good point! I've fixed that in 78f073165